syft/pkg/cataloger/python/test-fixtures/poetry/dev-deps/poetry.lock

```diff
@@ -11,6 +11,12 @@ docutils = "*"
 natsort = "*"
 six = "*"
 sphinx = "*"
+packaging = "*"
+msal = ">=0.4.1,<2.0.0"
+portalocker = [
+    {version = ">=1.0,<3", markers = "platform_system != \"Windows\""},
+    {version = ">=1.6,<3", markers = "platform_system == \"Windows\""},
+]
```

Comment:
we have a test case for a `string` value a `[]obj`, but not for just `obj` -- since the parsing code handles this case can you add one more dependency example in this fixture to make certain it's covered.

Also, it would be ideal to have a poetry.lock that is malformed in some way (e.g. `portalocker = [ [ [ { version = "1.2" }] ] ] )`) to show that we now survive this kind of parsing error

Reply:
@wagoodman I don't we we do survive this with the parser.

The parser will return an error if we encounter a malformed poetry file.
```
failed to read poetry lock package: toml: line 10 (last key "package.dependencies"): expected a top-level item to end with a newline, comment, or EOF, but got ')' instead
```

Do you want to assert that the parser is returning this error?

Reply:
Correction:
```
portalocker = [ [ [ { version = "1.2" } ] ] ] 
```